CITATION.cff

```diff
@@ -0,0 +1,52 @@
+# doc: https://citation-file-format.github.io/
+# schema: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+cff-version: 1.2.0
+
+title: "neurobagel cli"
+
+version: 0.2.0
+
+abstract: "The Bagel CLI is a simple Python command line tool to automatically read and annotate a [BIDS dataset](https://bids-specification.readthedocs.io/en/stable/) so that it can be integrated into the Neurobagel graph."
```

Comment:
hyperlinks don't seem to work properly in Zenodo descriptions - worth removing the BIDS spec URL?

Reply:
I think they want HTML style links, not markdown

Reply:
turns out they also don't like HTML links